### PR TITLE
[ENH] Move starter kit to new BIDS website with redirects

### PR DESCRIPTION
**Pull request by Remi-Gau**
Extract all pages and moves them to the new BIDS websites.

Each pages contains an explicit redirect for now but the TOC should point directly to the same page as well.

### CODE REVIEW

**Comment by bendhouseart** [comment]
So much change....is this ready for review?

**Comment by Remi-Gau** [comment]
not sure...

starting to get cold feet and if we should not do this across several PR rather than going nuclear like this...

or maybe keep this one open until we have more feedback on the new website...

**Comment by bendhouseart** [comment]
> not sure...
> 
> starting to get cold feet and if we should not do this across several PR rather than going nuclear like this...
> 
> or maybe keep this one open until we have more feedback on the new website...

I think the later given it's a Friday and I'm not confident there being much help to fix this with OHBM about to kick off. I would rather focus on what's currently working than breaking a bunch of new stuff this close to our website's "debut".

**Comment by bendhouseart** [comment]
I think @iamdamion would agree too.

**Comment by iamdamion** [comment]
Yeah, agree. Probably better to be safe for OHBM.

File: src/folders_and_files/derivatives.md [hunk 1]
```diff
@@ -1,234 +1,3 @@
...
-  raw_data/
-    sub-01/
-    ...
-  derivatives/
-    preprocessed/
-    analysis/
-```
-
-<!-- TODO derivatives JSON -->
+Moved [here](https://bids-website.readthedocs.io/en/latest/starter_kit/src/folders_and_files/derivatives.html)
```
Comment (Remi-Gau) [inline src/folders_and_files/derivatives.md:3]: example of explicit redirect

**Pull request merged**